HeterogeneousCore/SonicTriton/data/models/particlenet/config.pbtxt

```diff
@@ -0,0 +1,43 @@
+name: "particlenet"
```

Comment:
the files in `HeterogeneousCore/SonicTriton/data/models/` should be moved into https://github.com/fastmachinelearning/sonic-models (and removed from this branch)

Reply:
I removed them and moved them to: https://github.com/wpmccormack/sonic-models/tree/add_particlenet.  Got a PR waiting to put the model files in the main branch

Reply:
for reference, that PR is https://github.com/fastmachinelearning/sonic-models/pull/2